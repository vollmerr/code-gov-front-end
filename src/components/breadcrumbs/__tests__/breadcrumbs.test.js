import React from 'react';

import Breadcrumbs from 'components/breadcrumbs'
import { shallow } from 'enzyme';

describe('Testing <Breadcrumbs /> component', () => {
  const props = [{
    text: 'test',
    to: '/'
  }];
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Breadcrumbs crumbs={props} />);
  });
  it('renders Breadcrumb', () => {
    expect(wrapper.exists()).toEqual(true);
  })
  it('renders Breadcrumb without to', () => {
    const props = [{
      text: 'test',
    }];
    wrapper = shallow(<Breadcrumbs crumbs={props} />);
    expect(wrapper.exists()).toEqual(true);
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })
})