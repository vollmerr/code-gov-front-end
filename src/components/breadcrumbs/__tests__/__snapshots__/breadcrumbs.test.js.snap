// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Breadcrumbs /> component should render correct 1`] = `
<div
  className="indented"
>
  <ul
    className="breadcrumbs"
  >
    <li
      key="test"
    >
      <Connect(CustomLinkComponent)
        to="/"
      >
        test
      </Connect(CustomLinkComponent)>
    </li>
  </ul>
</div>
`;
